Marketing: Add Activitypub card

obenland opened the pull request:
<!--
Link a related Github/Linear issue to this PR. If the PR does not immediately resolve the issue,
for example, it requires a separate deployment to production, avoid
using the "fixes" keyword. Consider using "part of" instead.
-->
<img width="1510" alt="Screenshot 2025-07-01 at 1 55 50 PM" src="https://github.com/user-attachments/assets/503eacb8-3b49-4549-89a9-0865b7b875bf" />


Part of #

## Proposed Changes

* Adds Activitypub card in Marketing Tools
* Adds ActivityPub logo svg.

## Why are these changes being made?
<!--
It's easy to see what a PR does but much harder to find out why it was made,
particularly when researching old changes in history. Record an explanation of
the motivation behind this change and how it will help.
-->

* It's the age-old problem of discoverability. This is meant to make it easier for folks to discover ActivityPub and activate it.

## Testing Instructions

<!--
Add as many details as possible to help others reproduce the issue and test the fix.
"Before / After" screenshots can also be very helpful when the change is visual.
-->

* Checkout this PR and run `yarn && yarn start`.
* Go to http://calypso.localhost:3000/marketing/tools/ and select a simple site.
* Click "Enable ActivityPub" and make sure you get redirected to /marketing/connections.

## Pre-merge Checklist

<!--
Complete applicable items on this checklist **before** merging into trunk. Inapplicable items can be left unchecked.

Both the PR author and reviewer are responsible for ensuring the checklist is completed.
-->

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [x] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [ ] Have you tested accessibility for your changes? Ensure the feature remains usable with various user agents (e.g., browsers), interfaces (e.g., keyboard navigation), and assistive technologies (e.g., screen readers) (PCYsg-S3g-p2).
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
    - [ ] For UI changes, have we tested the change in various languages (for example, ES, PT, FR, or DE)? The length of text and words vary significantly between languages. 
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

obenland committed:
Marketing: Add Activitypub card

obenland commented:
We should probably use something like "Join the Social Web" instead of the jargon-y "Enable ActivityPub".
Ideally this should also just activate it and redirect to the Onboarding checklist (185367-ghe-Automattic/wpcom) when it's available.

obenland commented:
Looks like that duplicated Fiverr Logo has been around since #93426.

pfefferle commented:
I am not sure about the title. Is it really about "sharing the blog"? Isn't it more like sharing the content?

pfefferle commented on client/sites/marketing/tools/marketing-features-data.ts:
```diff
@@ -69,6 +70,21 @@ export const getMarketingFeaturesData = (
 			},
 		},
+		{
+			title: translate( 'Share your blog with a new audience' ),
```
```suggestion
			title: translate( 'Share your content with a new audience' ),
```

epeicher reviewed (approved):
I have tested it, and it works as expected. I can see the new card there and the link works as described. LGTM! 

|WP Cloud site | Simple site | Clicking the link|
|-------|------|------|
| ![CleanShot 2025-07-02 at 10 06 04@2x](https://github.com/user-attachments/assets/77e8caa8-2709-4bc7-93d2-b24d1d90da53)|![CleanShot 2025-07-02 at 10 09 29@2x](https://github.com/user-attachments/assets/e929d624-b152-4186-adfc-bf75996089b0)| ![CleanShot 2025-07-02 at 10 06 30@2x](https://github.com/user-attachments/assets/1574df6a-a105-41d8-bf89-ff15fb65c679)|

obenland committed:
Update ActivityPub button text and tracking event

Changed the ActivityPub feature button text to 'Enter the fediverse' and updated the tracking event name for more accurate analytics.

obenland committed:
Merge branch 'trunk' into add/activitypub-card

a8ci18n commented:
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/19457085

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @obenland for including a screenshot in the description! This is really helpful for our translators.
